scripts/start_server.sh

```diff
@@ -28,7 +28,7 @@ gunicorn --bind :8080 --workers 4 main:app
 #  $BASEDIR/../dispatch.yaml \
 #  $BASEDIR/../notifier.yaml \
 #  $BASEDIR/../dev-default.yaml \
```

Comment:
is `dev-default.yaml` still needed? We can delete the file at this point.

Reply:
Good point.  Gone.